tests/baselines/reference/tsxStatelessFunctionComponentOverload4.errors.txt

```diff
@@ -82,4 +83,7 @@ tests/cases/conformance/jsx/file.tsx(35,29): error TS2322: Type 'string' is not
 !!! error TS2322: Type 'string' is not assignable to type 'boolean'.
 !!! related TS6500 tests/cases/conformance/jsx/file.tsx:30:38: The expected type comes from property 'y1' which is declared here on type 'IntrinsicAttributes & { y1: boolean; y2?: number; y3: boolean; }'
     const e4 = <TestingOptional y1="hello" y2={1000}>Hi</TestingOptional>
+                                ~~
+!!! error TS2322: Type 'string' is not assignable to type 'boolean'.

```

Comment:
You might be thinking "why is there a new error here?" well, because the signature this _was_ compatible with is now eliminated by the excess child prop, and a `string` child prop isn't compatible with any of the other signatures~

Reply:
Oh, wow, bonus bugfix 😄

The error is definitely confusing, though, but such is how overload resolution goes... it's always on the last overload, not on the "almost, but not quite, more specific"